utils/log.go

```diff
@@ -1,16 +1,12 @@
 package utils
 
 import (
-	"github.com/pengsrc/go-shared/check"
-	"github.com/pengsrc/go-shared/log"
+	"go.uber.org/zap"
 )
 
-// Logger is the global logger for BeyondFTP
-var Logger *log.ContextFreeLogger
-
-func init() {
+func SetUpLog() {
```

Comment:
I suggest that we can add a local package named `logger`, and group several actions together, such as parse log level, set up logger, and so on.

Reply:
ditto.